tools/kafkatool/consumer_group.go

```diff
@@ -90,6 +95,36 @@ func (c *ConsumerGroupCommand) copyOffset(_ *kingpin.ParseContext) error {
 	return commitConsumerGroupOffset(adm, c.copyToGroup, c.topic, c.partitionID, offset.At, c.printer)
 }
 
+func (c *ConsumerGroupCommand) deleteOffsets(_ *kingpin.ParseContext) error {
+	client := c.getKafkaClient()
+	adm := kadm.NewClient(client)
+
+	return deleteConsumerGroupOffsets(adm, c.group, c.topic, c.partitionID, c.printer)
+}
+
+func deleteConsumerGroupOffsets(adm *kadm.Client, group, topic string, partitionID int32, printer Printer) error {
+	toDelete := kadm.TopicsSet{}
+	toDelete.Add(topic, partitionID)
+
+	results, err := adm.DeleteOffsets(context.Background(), group, toDelete)
+	if err != nil {
+		return err
+	}
+
+	err, ok := results.Lookup(topic, partitionID)
```

Comment:
I manually tested it against Warpstream and I got the error `unable to determine the result of the delete offsets operations for...` but the deletion worked. The reason why I got the error is because `results` only contains errors. If there's no error, then the lookup will find nothing, but it doesn't mean it was not successful. You should just remove the `if` branch checking for `ok`.